braze/src/main/java/com/rudderstack/android/integration/braze/BrazeIntegrationFactory.java

```diff
@@ -93,23 +94,23 @@ private BrazeIntegrationFactory(Object config, final RudderClient client, Rudder
             if (destinationConfig.containsKey(AUTO_IN_APP_MESSAGE_REGISTER)) {
                 String autoInAppMessageRegEnabledStr = (String) destinationConfig.get(AUTO_IN_APP_MESSAGE_REGISTER);
                 if (autoInAppMessageRegEnabledStr != null) {
-                    autoInAppMessageRegEnabled = Boolean.getBoolean(autoInAppMessageRegEnabledStr);
+                    this.autoInAppMessageRegEnabled = Boolean.getBoolean(autoInAppMessageRegEnabledStr);
                 }
             }
 
             // get endpoint
-            String endPoint = "";
+            String endPoint = null, customEndPoint = null;
             if (destinationConfig.containsKey(DATA_CENTER_KEY)) {
                 endPoint = (String) destinationConfig.get(DATA_CENTER_KEY);
             }
 
-            if (TextUtils.isEmpty(endPoint)) {
+            if (endPoint == null || endPoint.isEmpty()) {
```

Comment:
Why did we replace `TextUtils.isEmpty()` with explicit checks?

Reply:
Resolving as discussed